.bit/tests/test.1.8.js

```diff
@@ -34,6 +34,28 @@ try {
             console.log(`YIKES! We got ${data} instead of ${answer}. Try again!`)
             process.exit(1)
         }
+
+        try {
```

Comment:
I would also try adding a `fetch(uri + "&plaintext="`) to check if they have entered the `plaintext` parameter, but not a value for `plaintext`

Reply:
Correct me if I'm wrong, but I'm pretty sure sending a blank parameter and not sending one at all is the same thing.

Reply:
It depends on what we are doing with it. If you use the param to join to a string it will give different results

```
function test(param1)
{
  console.log("Testing..." + param1);
}


test()
test('')
test(null)
```